src/utils/components/CustomDialog.vue

```diff
@@ -3,14 +3,17 @@
     v-bind="$attrs"
     v-on="$listeners"
   >
-    <QCard class="q-dialog-plugin">
+    <QCard
+      class="q-dialog-plugin"
+      :style="{ width: $attrs.width ? $attrs.width : ''}"
+    >
       <QCardSection class="q-dialog__title">
         <slot name="title" />
       </QCardSection>
       <QCardSection class="q-dialog__message">
         <slot name="message" />
       </QCardSection>
-      <QCardActions align="right">
+      <QCardActions :align="$attrs.actionsalign ? $attrs.actionsalign : 'right'">
```

Comment:
Follow up to the nitpicky point, if possible these could use `actionsAlign` casing.